docs/block-processing/slog.md

```diff
@@ -16,10 +16,10 @@ Slog is responsible for the following (see explanations below):
     -   `inMainChain` flags
 -   Calling callbacks (`BListener`)
 
-Pure verification is a set of checks in `Pos.Block.Pure` that operate on
-arbitrary sequences of blocks and verify in a pure context whether these blocks
-constitute a valid chain. For more information, see Haddock comments in
-`Pos.Block.Pure`
+Pure verification is a set of checks in `Pos.Block.Logic.Integrity`
```

Comment:
Please reflow text with the following command:

```
pandoc --from=markdown --to=markdown_github+yaml_metadata_block-hard_line_breaks --columns=80 --atx-headers --normalize $1 -o $1
```

where `$1` is the filename (`slog.md`).